### DISCUSSION TITLE
Invalid initialization vector

### DISCUSSION THREAD

**Discussion by Gp2mv3** [Budibase/budibase#14458] [Help]
**Checklist**
- [V] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**
- Self
  - Method: Docker compose
  - Budibase Version: latest
  - App Version: latest
 
**Describe the bug**
When trying to login, I have an error "Invalid initialization vector".

Here is the stacktrace from the bbworker:
> bbworker              | {"level":"WARN","timestamp":"2024-08-24T10:11:02.738Z","service":"@budibase/worker","err":{"type":"TypeError","message":"Invalid initialization vector","stack":"TypeError: Invalid initialization vector\n    at Decipheriv.createCipherBase (node:internal/crypto/cipher:121:19)\n    at Decipheriv.createCipherWithIV (node:internal/crypto/cipher:140:3)\n    at new Decipheriv (node:internal/crypto/cipher:289:3)\n    at Object.createDecipheriv (node:crypto:155:10)\n    at decrypt (/app/dist/index.js:135589:43)\n    at checkApiKey (/app/dist/index.js:135719:21)\n    at /app/dist/index.js:135793:50\n    at dispatch (/app/node_modules/koa-compose/index.js:42:32)\n    at /app/node_modules/@koa/router/lib/router.js:368:16\n    at dispatch (/app/node_modules/koa-compose/index.js:42:32)\n    at /app/node_modules/koa-compress/lib/index.js:38:11\n    at dispatch (/app/node_modules/koa-compose/index.js:42:32)\n    at /app/node_modules/@koa/router/lib/router.js:368:16\n    at dispatch (/app/node_modules/koa-compose/index.js:42:32)\n    at errorHandling (/app/dist/index.js:140306:11)\n    at dispatch (/app/node_modules/koa-compose/index.js:42:32)","code":"ERR_CRYPTO_INVALID_IV","expose":true,"statusCode":403,"status":403},"pid":28,"tenantId":"default","correlationId":"2a3a8baa-cf00-41cb-9ab9-debdb5459426","msg":""}

**To Reproduce**
1. Follow the install method of the doc for docker-compose
2. Run it
3. Try to connect
5. See error in log, and you're redirected to the login screen

**Expected behavior**
Being able to login

**Additional context**
My installation is behind a reverse proxy (swag, so nginx). But I don't think it matters as the error is from nodeJS.

**Comment by Gp2mv3** [comment]
Why did you convert it to a discussion ? Isn't it an issue ? Also, is it possible to access the discussion on Linear ? I don't seem to have access to the workspace.

**Reply by shogunpurple** [reply]
Hey @Gp2mv3 

- We don't consider it an issue until we can reliably recreate it. The standard docker-compose setup works - this issue is likely to be environmental. What OS are you using, and what version of NodeJS are you on?
- Linear is what we use internally for managing issues. Only the team have access to it - it syncs back to github though, so the community can see updates

**Comment by AdFisch** [comment]
Hi @Gp2mv3,

I'm encountering the same error so I was wondering if you've ever found a solution?

Thank you. :-)
